.github/workflows/ci.yaml

```diff
@@ -54,10 +60,66 @@ jobs:
       TRIAL_BRANCH: ${{ needs.metadata.outputs.TRIAL_BRANCH }}
     secrets: inherit
 
+  publish-mealkit:
+    needs: [metadata, amd64, arm64]
+    if: "!canceled()" && github.event_name == 'schedule' && needs.metadata.outputs.PUBLISH == 'true'
```

Comment:
`github.event_name == 'schedule' ` looks like it's already captured in `needs.metadata.outputs.PUBLISH`